tests/Application/config/packages/doctrine.yaml

```diff
@@ -1,23 +0,0 @@
-parameters:
-    # Adds a fallback DATABASE_URL if the env var is not set.
-    # This allows you to run cache:warmup even if your
-    # environment variables are not available yet.
-    # You should not need to change this value.
-    env(DATABASE_URL): ''
-
-doctrine:
-    dbal:
-        driver: 'pdo_mysql'
-        charset: utf8mb4
```

Comment:
I'm pretty sure you are removing something interesting around this line.

Reply:
Is there any possibility to use a symlink for this file as well? Isn't it easier?